packages/backend-modules/auth/graphql/schema-types.js

```diff
@@ -47,6 +47,7 @@ type User {
   # true: consented
   # false: consent revoked
   hasConsentedTo(name: String!): Boolean
+  alias: String!
```

Comment:
Should the alias rather be in the referral-campaign package? Or do you think it's a generic alias that we might also use for other features?

Reply:
If generic, should it rather be named something like `shortId` which aligns more with our intent? `alias` sounds more like something users can choose for themselves.